### DISCUSSION TITLE
Bulk Insertion of Master Data from serverpod

### DISCUSSION THREAD

**Discussion by ashleshacc** [serverpod/serverpod#587] [Q&A] [answered]
Hi @vlidholt @yahu1031 , i wanted to ask what is the best way to bulk insert data into tables which i make using serverpod, if want to bulk insert data into those tables.

**Comment by vlidholt** [comment, marked as answer]
It depends on your use case. If you need to insert many rows at once when the server is running, your best option may be to do a custom query over the regular insert method.

If you are importing data from another source, you may want to work directly against the database using the psql command line tool.
